tests/rws_tests.py

```diff
@@ -1062,6 +1065,7 @@ def test_robots_none_tag(self, mock_get):
         loaded_sets = rws_check.load_sets()
         rws_check.find_robots_txt(loaded_sets)
         self.assertEqual(rws_check.error_list, [])
+        mock_get.assert_called_once_with('https://service4.com', timeout=10, allow_redirects=False)
 
```

Comment:
Instead of inserting the calls to `assert_called_once_with` into the pre-existing robots tests, I would prefer a single test case that would produce a different result if not called with `allow_redirects=False`, such as adding the following case to line 924: 
```
    elif args[0] == 'https://service7.com':
        if 'allow_redirects' in kwargs:
            if kwargs['allow_redirects'] != True:
                return MockedGetResponse({"X-Robots-Tag":"noindex"}, 200)
        return MockedGetResponse({"X-Robots-Tag":"foo"}, 200) 
```
And inserting the following test here:

```suggestion
    
    @mock.patch('requests.get', side_effect=mock_get)
    def test_robots_redirects(self, mock_get):
        json_dict = {
            "sets":
            [
                {
                    "primary": "https://primary.com",
                    "serviceSites": ["https://service7.com"]
                }
            ]
        }
        rws_check = RwsCheck(rws_sites=json_dict,
                     etlds=None,
                     icanns=set())
        loaded_sets = rws_check.load_sets()
        rws_check.find_robots_txt(loaded_sets)
        self.assertEqual(rws_check.error_list, [])
        
```

Reply:
Ack. Just made these changes. Thanks for the review.